armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -64,26 +70,25 @@ def expandColdDimsToHot(assems, isDetailedAxialExpansion, referenceAssembly=None
     assems: list[:py:class:`Assembly <armi.reactor.assemblies.Assembly>`]
         list of assemblies to be thermally expanded
     isDetailedAxialExpansion: bool
-        If true assemblies will be forced to conform to the reference mesh after expansion
+        If False, assemblies will be forced to conform to the reference mesh after expansion
+    assemsToSkip: List[str]
+        list of strings to be converted to flags to indicate which assemblies to skip axial expansion
```

Comment:
It would seem better to me to just pass the flags directly. That way someone who has the flags doesn't need to de-convert them into strings in order to use this function.

Reply:
Addressed https://github.com/terrapower/armi/pull/1235/commits/32ce614201e443c1a4e6db86c634f16c0f8856ad